File: AndroidStudioProjects/LoginRetrofit/app/src/main/java/com/example/loginretrofit/MainActivity.java
package com.example.loginretrofit;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import com.example.loginretrofit.api.ApiInterface;
import com.example.loginretrofit.api.ApiUtil;
import com.example.loginretrofit.model.RegisterData;

import retrofit2.Call;
import retrofit2.Callback;
import retrofit2.Response;

public class MainActivity extends AppCompatActivity {

    ApiInterface apiInterface;

    EditText userName,email,password,passwordConfirm;
    Button buttonRegister;
    TextView txtresponse;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        userName=findViewById(R.id.name);
        email=findViewById(R.id.email);
        password=findViewById(R.id.password);
        passwordConfirm=findViewById(R.id.confirm_password);

        buttonRegister=findViewById(R.id.register);

        txtresponse=findViewById(R.id.response);

        buttonRegister.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                register();
            }
        });


    }
    public void register(){
        String name=userName.getText().toString().trim();
        String userEmail=email.getText().toString().trim();
        String userPassword=password.getText().toString().trim();
        String passwordConfirmmation=passwordConfirm.getText().toString().trim();
        apiInterface= ApiUtil.getApi();

        Call<RegisterData> call=apiInterface.registerUser(name,userEmail,userPassword,passwordConfirmmation);

        call.enqueue(new Callback<RegisterData>() {
            @Override
            public void onResponse(Call<RegisterData> call, Response<RegisterData> response) {
                if (response.isSuccessful()){
                    RegisterData registerData=response.body();
                    txtresponse.setText(response.body().getMessage());
                }
            }

            @Override
            public void onFailure(Call<RegisterData> call, Throwable t) {
                Toast.makeText(MainActivity.this,t.toString(),Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: AndroidStudioProjects/MobileApp/app/src/main/java/com/example/mobileapp/MainActivity.kt
package com.example.mobileapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val edtNumOne=findViewById<EditText>(R.id.edit_num_one)
        val edtNumTwo=findViewById<EditText>(R.id.edit_num_two)

        val btnSum:Button=findViewById(R.id.btn_sum)
        val btnSub:Button=findViewById(R.id.btn_sub)
        val btnMul:Button=findViewById(R.id.btn_mul)

        val txtResult:TextView=findViewById(R.id.txt_result)

        btnSum.setOnClickListener {
            var numOne:Int=edtNumOne.text.toString().toInt()
            var numTwo:Int=edtNumTwo.text.toString().toInt()

            var result=numOne+numTwo
            txtResult.text=result.toString()
        }

        btn_sub.setOnClickListener {
            var numOne:Int=edtNumOne.text.toString().toInt()
            var numTwo:Int=edtNumTwo.text.toString().toInt()

            var result=numOne-numTwo
            txtResult.text=result.toString()
        }

        btnMul.setOnClickListener {
            var numOne:Int=edtNumOne.text.toString().toInt()
            var numTwo:Int=edtNumTwo.text.toString().toInt()

            var result=numOne*numTwo
            txtResult.text=result.toString()
        }




    }
}
